modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -9,47 +9,46 @@ A message type in Partner Manager can be used in one of the following ways:
 * Source at host
 * Send to partners
 
-Message types are always managed within the host partner profile page, regardless of whether they are used specifically for a single partner or across multiple trading partners. Message types are reusable across message flows for different trading partners.
+Message types are always managed in the host profile page, regardless of whether they are used for a single partner or for multiple trading partners. Additionally, message types are reusable across message flows for different trading partners.
 
-Create message types from the Host page or from the message flow configuration wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
+You can create message types from the Host page or from the Message Flow wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
 
 == Create a Message Type
 
-. In Partner Manager left navigation, select *<host-name>*. 
+. In Partner Manager left navigation, select *<host-name>*.
 . From the *Message types* section host page:
-. Click *New*.
-. In the *Usage* drop-down list, select the message type use:
+.. Click *New*.
+..  In the *Usage* drop-down list, select the message type:
 * <<receive-and-send-to-partners,*Receive from Partners*>>
 * <<target-at-host,*Target at host*>>
 * <<source-at-host,*Source at host*>>
 * <<receive-and-send-to-partners,*Send to partners*>>
 +
-If you create a message type from the message flow, Partner Manager automatically designates the *Usage* value. 
-. Configure the message type according to the usage you selected.
+If you create a message type from the message flow, Partner Manager automatically determines the *Usage* value.
 . Click *Save*. +
 The message type appears in the *Message types* section on the host page.
 
 [[receive-and-send-to-partners]]
 == Receive From and Send To Partners
```

Comment:
Is this part of Configure?
Hmm...I think not create or configure, which are tasks, but a SEPARATE conceptual topic, Message Type Descriptions.

Reply:
Updated new story for this change